1 HW_Standard/HW_Sunflora.yaml

```diff
@@ -20,3 +20,26 @@ nature: Naive
 ivs: {hp: 31, atk: 31, def: 31, spA: 31, spD: 31, spe: 31}
 evs: {hp: 96, atk: 56, def: 0, spA: 252, spD: 0, spe: 252}
 hidden: false
+---
+species: Sunflora
+ingamename: Stray Cat
```

Comment:
why is a flower a stray cat? am i missing something?

Reply:
[It’s a reference.](https://jojo.fandom.com/wiki/Stray_Cat)